docs/shared/guides/module-federation/module-federation-and-nx.md

```diff
@@ -0,0 +1,126 @@
+
+{% callout type="warning" title="Release Cadences" %}
+Despite allowing for each team to adhere to its own release cadence, there are some changes that should still be co-ordinated with teams, such as package upgrades, as this can lead to an incompatibility and unexpected behaviour between the applications within the Module Federation Architecture.
+
+For example, upgrading Angular or React to the latest version should be co-ordinated to prevent any issues where different versions are loaded at runtime, causing unexpected behaviour.
+{% /callout %}
+
+In order to achieve this, Module Federation introduces three terms for the applications that make up the Module Federation architecture; `host`, `remote` and `federated modules`.
+
+### What is a Remote?
+
+A `remote` is an application that exposes a federated module that can be fetched over the network at runtime. The federated module can be any valid JavaScript module, and therefore grows to include things such as a React Component, an Angular Routing File, a plain old JavaScript object (POJO) and more.
+
+{% callout type="note" title="Creating a Remote" %}
+
+Follow our [How to Create a Remote Application Recipe](/recipes/module-federation/create-a-remote) to learn more.
+
+{% /callout %}
+
+### What is a Host?
+
+A `host` is an application that consumes federated modules from `remote` applications at runtime.
+
+When you write your host application, you import the module from your remote as though it was part of the build, but at build time, Webpack is aware that this module will only exist at runtime, and only after it has made a network request to the corresponding remote application to fetch the JS bundle.
+
+The federated module will then be executed as though it was always part of the `host` application.
+
+{% callout type="note" title="Creating a Host" %}
+
+Follow our [How to Create a Host Application Recipe](/recipes/module-federation/create-a-host) to learn more.
+
+{% /callout %}
+
+### What is a Federated Module?
+
+A `federated module` is any valid JavaScript module that is exposed by a `remote` application with the aim that it will be consumed by a `host` application.
+
+This means that React Components, Angular Components, Services, Application State, Functions, UI Components and more can be shared between applications and updated without the need to redeploy everything.
+
+{% callout type="note" title="Federating a Module" %}
+
+Follow our [How to Federate a Module Recipe](/recipes/module-federation/federate-a-module) to learn more.
+
+{% /callout %}
+
+### Common Pitfalls
+
+Module Federation is not without its complexity, especially if you choose to use it to enable independent deployments where independent `remotes` are deployed on different release cadences. Some of the pitfalls you might encounter are listed below:
+
+#### Increased bundle size
```

Comment:
Consistent casing, I think we want to title case everything. `Increased Bundle Size`